css/avatar.css

```diff
@@ -0,0 +1,87 @@
+/* @import url("https://ripple-ui.netlify.app/css/components.css"); */
+@import url("../../../../css/components.css");
+body {
+	font-family: sans-serif;
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+}
+.avatar-container {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	gap: 1rem;
+}
+
+.avatar {
+	border-radius: 50%;
+}
+
+.avatar-text {
+	display: flex;
+	justify-content: center;
+	background: var(--cta-color);
+	align-items: center;
+	font-weight: bold;
+}
+.xl {
+	height: 8rem;
+	width: 8rem;
+}
+
+.l {
+	height: 5rem;
+	width: 5rem;
+}
+
+.m {
+	height: 3rem;
+	width: 3rem;
+}
+.s {
+	height: 2rem;
+	width: 2rem;
+}
+.avatar-badge {
+	height: 1.5rem;
+	width: 1.5rem;
+	border: 0.2rem solid var(--secondary-color);
+	border-radius: 50%;
+	position: relative;
+}
+
+.avatar-badge-xl {
+	top: -2.5rem;
+	right: -6rem;
+}
+
+.avatar-badge-l {
+	top: -2rem;
+	right: -3.6rem;
+}
+
+.avatar-badge-m {
+	width: 1rem;
+	height: 1rem;
+	border: 0.2rem solid var(--secondary-color);
+	top: -1.5rem;
+	right: -2rem;
+}
+.avatar-badge-s {
+	width: 0.8rem;
+	height: 0.8rem;
+	border: 0.1rem solid var(--secondary-color);
+	top: -1rem;
+	right: -1.5rem;
+}
+
+.online {
+	background: var(--online-badge-color);
+}
+.away {
+	background: var(--away-badge-color);
+}
+
+.busy {
+	background: var(--busy-badge-color);
+}
```

Comment:
this file looks like a duplicate of components/avatars/src/css/avatar.css, Check if both are required.

Reply:
yes, I am including all the CSS files inside that folder so when a user needs to add a particular CSS file he can add that form here itself instead of going into that folder. But I think I can delete the file present inside after the app is deployed and replace it with import from CSS folder

Reply:
yes, you can use the same css file anywhere you want. keeping the project DRY is important

Reply:
ok